optimum/habana/transformers/training_args.py

```diff
@@ -118,6 +118,8 @@ class GaudiTrainingArguments(TrainingArguments):
             Number of steps to ignore for profling.
         profiling_steps (`int`, *optional*, defaults to 0):
             Number of steps to be captured when enabling profiling.
+        profiling_record_shapes (`bool`, *optional*, defaults to True):
+            Record shapes when enabling profiling.
```

Comment:
I don't think we should add this as there is no new argument defined in `training_args.py.

Reply:
where should we add this?

Reply:
IT's already added in the docstrings of `generation/utils.py` and in the help section of the new arg of the text-generation example. This is good enough.

Reply:
removed